src/flat-button.jsx

```diff
@@ -197,7 +258,7 @@ const FlatButton = React.createClass({
         onKeyboardFocus={this._handleKeyboardFocus}
         onMouseLeave={this._handleMouseLeave}
         onMouseEnter={this._handleMouseEnter}
-        onTouchStart={this._handleTouchStart}
```

Comment:
@oliviertassinari I see, so then I won't see it when I try it... Ok. I will leave that prop there for now.